### PR TITLE
Add Deux Huit Huit

### PR DESCRIPTION
**Pull request by nitriques**
Thanks! We :heart: open source and Montréal !

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,6 +58,7 @@ These projects have been developed mainly in Montréal, or by Montréalers.
 - [@BetterMTL](https://github.com/bettermtl) - Data layered to engage city-wide activities and teams. [Website](https://bettermtl.github.io/).
 - [@busbud](https://github.com/busbud) [Website](https://www.busbud.com/).
 - [@ctf](https://github.com/ctf) - McGill Science Computer Taskforce. [Website](https://ctf.science.mcgill.ca/)
+- [@DeuxHuitHuit](https://github.com/DeuxHuitHuit) - Deux Huit Huit [Website](https://deuxhuithuit.com/)
```
Comment (khos2ow) [inline README.md]: Our Travis does a health check on URLs, and it fails if there's a `30x` HTTP status, which is the case for your site. It seems you have a redirect to `https://deuxhuithuit.com/fr/`. Can you adjust this please?

Comment (nitriques) [reply README.md]: It would redirect you to /en/ on an english browser... Is there a way to add an exception for this ?

Comment (khos2ow) [reply README.md]: Unfortunately we cannot do that. @RichardLitt what's your take on this?

Comment (nitriques) [reply README.md]: We are a bilingual city... aren't we ?

Comment (khos2ow) [reply README.md]: We are, but Travis isn't. But I think I've come up with a solution!

For checking the integrity of the links we can let Travis to fail the build for every `30x` statuses except for `302`, which technically will be used for such cases and technically they are not disruptive.

Comment (nitriques) [reply README.md]: Yé!

Comment (RichardLitt) [reply README.md]: That should work. We've got to allow bilingualism. :)

**Comment by RichardLitt** [comment]
Thank you, @nitriques! This should work now. :)

**Comment by nitriques** [comment]
@RichardLitt Thanks !

**Pull request merged**